packages/mobile/src/home/WalletHome.tsx

```diff
@@ -110,10 +101,17 @@ function WalletHome() {
   }
 
   const shouldShowCashInBottomSheet = () => {
-    const hasStable = STABLE_CURRENCIES.some((currency) =>
-      balances[currency]?.isGreaterThan(STABLE_TRANSACTION_MIN_AMOUNT)
+    // If there are no core tokens then we are either still loading or loading failed.
+    if (coreTokenBalances.length < 2) {
```

Comment:
any reason why this condition is less than 2 and not 1?

Reply:
Well, if there's only one core token either `hasStable` or `hasCelo` will return `false` and the whole function will return `false`, but I guess it's not clear from this so I'll change it :)